sklearn/preprocessing/data.py

```diff
@@ -47,41 +50,87 @@
 ]
 
 
-def _mean_and_std(X, axis=0, with_mean=True, with_std=True):
-    """Compute mean and std deviation for centering, scaling.
+def _incremental_mean_and_var(new_X, axis=0, last_mean=.0, last_var=None,
+                              last_n=0):
```

Comment:
I would rename `last_n` to `last_n_samples_seen` to be more explicit and more consistent.